module6/hangman_game.rb

```diff
@@ -0,0 +1,116 @@
+def hangman(word)
+  lifes = 5;
+  incomplete_word = Array.new(word.length, "_")  #new aaray of word size                          
+  while lifes > 0  do
+    puts "lifes remaining #{lifes}"     #printing remaining lifes
+    puts incomplete_word.join(" ");          #printing word with blank spaces
+    puts 'Enter the letter : '
+    letter = gets.chomp.downcase       
+    if(letter.length != 1)
+      puts 'enter only one letter at a time'  #if user enters more than one letter
+      next
+    else
+      if word.include? letter      #checking if entered letter is present in the reamining letters
+        word.chars.each_with_index do |ch , index|   #loop for positioning the guessed character at perticular position
+          if ch == letter
+           incomplete_word[index] = ch      
+          end
+        end
+        if !incomplete_word.include? "_"            #if word is complete then break
+          break;
```

Comment:
You can avoid if end block by writing it inline as it is one line block:
break if  !incomplete_word.include? "_"